### PR TITLE
Schedule community.network for removal in Ansible 12

### PR DESCRIPTION
**Pull request by gotmax23**
Relates: https://forum.ansible.com/t/8030
Relates: https://github.com/ansible-community/ansible-build-data/issues/475

### CODE REVIEW

**Comment by gotmax23** [comment]
We still need to add the deprecation notices to the community.network collection, so this is a draft for now

**Review by felixfontein — APPROVED** [review]

**Comment by mariolenz** [comment]
#ref ansible-collections/community.network#618

File: 11/collection-meta.yaml [hunk 1]
```diff
@@ -128,6 +128,11 @@ collections:
     repository: https://github.com/ansible-collections/community.mysql
   community.network:
     repository: https://github.com/ansible-collections/community.network
+    removal:
+      major_version: 12
+      announce_version: 11.0.0a2
```
Comment (felixfontein) [inline 11/collection-meta.yaml:133]: Assuming there is no further alpha release, this needs to be updated to:
```suggestion
      announce_version: 11.0.0b1
```

**Comment by felixfontein** [comment]
Now that community.network is officially deprecated (with the 5.1.0 release), I think we can merge this.

**Comment by gotmax23** [comment]
Thanks. Feel free to merge once CI passes.

**Comment by felixfontein** [comment]
@gotmax23 @mariolenz thanks!

**Pull request merged**